### PR TITLE
standardize errors for feature service chain

### PR DESCRIPTION
**Pull request by dmfenton**
@ngoldman this is meant to work in sync with the featureservice PR

### CODE REVIEW

File: models/agol.js [hunk 1]
```diff
@@ -254,6 +266,46 @@ var AGOL = function (koop) {
   }
 
   /**
+   * Wraps Cache.updateInfo for easier testing
+   * @param {string} key - a table name in the db
+   * @param {object} info - the info to store in the table
+   * @param {function} callback  - calls back with err or nothing
+   */
+  agol.updateInfo = function (key, info, callback) {
+    koop.Cache.getInfo(key, info, callback)
```
Comment (mjuniper) [inline models/agol.js]: @dmfenton - I'm waaaay off the back on koop. But I'm not sure this is right. The comment says "Wraps Cache.updateInfo..." and the method is called `updateInfo` but you actually wrap `getInfo`.

Comment (dmfenton) [inline models/agol.js]: Thanks for spotting that. That's laziness in copy/pasting on the docs. Will correct.

Comment (dmfenton) [inline models/agol.js]: Actually this doesn't look right at all. @mjuniper++

**Comment by dmfenton** [comment]
ping @ngoldman this is ready for review

**Comment by ngoldman** [comment]
lgtm :+1:

**Pull request merged**